diff --git a/dist/index-D7ja2BKs.d.ts b/dist/index-D7ja2BKs.d.ts
index 9550c4d8d1ae7bf9868a83038e70ad9f9a2da054..3b89f8db380d79553818c39147d30211cb460eac 100644
--- a/dist/index-D7ja2BKs.d.ts
+++ b/dist/index-D7ja2BKs.d.ts
@@ -1196,11 +1196,11 @@ interface EXPERIMENTAL_ResolverRecord_Base {
    * Name of the matcher. Unique across all matchers. If missing, this record
    * cannot be matched. This is useful for grouping records.
    */
-  name?: RecordName;
+  name?: RecordName | undefined;
   /**
    * {@link MatcherPattern} for the path section of the URI.
    */
-  path?: MatcherPatternPath;
+  path?: MatcherPatternPath | undefined;
   /**
    * {@link MatcherPattern} for the query section of the URI.
    */
@@ -1208,7 +1208,7 @@ interface EXPERIMENTAL_ResolverRecord_Base {
   /**
    * {@link MatcherPattern} for the hash section of the URI.
    */
-  hash?: MatcherPatternHash;
+  hash?: MatcherPatternHash | undefined;
   /**
    * Parent record. The parent can be a group or a matchable record.
    * It will be included in the `matched` array of a resolved location.
